Polishing Sprites

FoeFear opened the pull request:
Updates the save menu icon sprites since they haven't been updated to contain the newer sprites

Other Changes:
-Ion Shooter sprite outlines are slightly wider
-Added Bg1 on the main menu since it was used in ep1 (is positioned below the title to prevent issues for people who are trying to read the title, hopefully)
-Fixed a translucent part on the square background sprite
-Polished the spawner and broken spawner sprites
-Fixed the small cut on the shard rock tier 2 sprite
-Polished a bunch of part sprites (all of them weren't technically changed by its lengths and widths, but the outlines were mainly updated. Plus, they use less space)

~~Additions:~~
~~-Added a audio pause mechanic where audio pauses when you pause the game (this would be more of a temporary thing)~~
~~-Healer tower bunker sprite was added (can be removed if requested)~~
Moving these two into another branch instead

rudderbucky commented:
* I quite like the line at the center of the spawners, why'd you remove it?
* What do you mean by "outlines were mainly updated"? Did you change the svgs or just the png files?

FoeFear commented:
For the spawner, that line in the center wasn't supposed to be there when you compare it to the original.

For the outlines, I updated the svgs (height and width were not changed) and converted them to pngs. It was more to remove the translucent sections between the outlines and filler colour.

FoeFear commented:
Closing it since the number of commits is messy. Going to make a better one.
